clang/Cursor.d

```diff
@@ -281,6 +281,11 @@ struct Cursor
         foreach (cursor; unit.cursor.children)
             result[cursor.spelling] = cursor.kind;
 
+        auto version_ = clangVersion();
+
+        if (version_.major == 3 && version_.minor == 7)
+            result["__int64"] = CXCursorKind.CXCursor_MacroDefinition;
```

Comment:
No the versions 3.4, 3.5, 3.6 are using MinGW too, but they work fine (which is a bit strange...).